src/Compilers/CSharp/Test/Semantic/Semantics/OutVarTests.cs

```diff
@@ -1457,6 +1494,1629 @@ class Test : System.Attribute
         }
 
         [Fact]
+        public void Scope_Catch_01()
+        {
+            var source =
+@"
+public class X
+{
+    public static void Main()
```

Comment:
I prefer to keep the commented code in the test source as placeholders for where we would wan't to test ```let``` declarations. This is consistent with pattern matching tests.